client/verta/tests/test_artifacts.py

```diff
@@ -88,6 +91,42 @@ def test_upload_dir(self, experiment_run, strs, dir_and_files):
         with zipfile.ZipFile(experiment_run.get_artifact(key), 'r') as zipf:
             assert filepaths == set(zipf.namelist())
 
+    @pytest.mark.not_oss
```

Comment:
Why not?

Reply:
Because multipart upload doesn't occur in our OSS setup, so that is what I'm using to determine whether or not to run this test.

Reply:
Why it doesn't occur?

Reply:
MDB doesn't use the multipart upload procedure in our OSS setup with NFS as its artifact store. It just gives a single URL to upload the whole artifact.

Reply:
Oh yeah, I forgot about that.